desktop-exporter/app/types/metadata-types.ts

```diff
@@ -1,11 +1,22 @@
 import { SpanData } from "./api-types";
 
-export type SpanMetaData = {
-  depth: number;
-};
+export enum SpanDataStatus {
+  missing,
+  present,
+}
```

Comment:
I forgot that TypeScript added enums like this to the language. Using it like this can create issues while debugging since logging out the object will log `0` and `1` for `status` instead of something human-readable.

The solution to that problem is to add string values to your enum like this:

```typescript
export enum SpanDataStatus {
  missing = "missing",
  present = "present",
}
```